pandas/core/series.py

```diff
@@ -1842,35 +1842,6 @@ def argsort(self, axis=0, kind='quicksort', order=None):
                 np.argsort(values, kind=kind), index=self.index,
                 dtype='int64').__finalize__(self)
 
-    def rank(self, method='average', na_option='keep', ascending=True,
-             pct=False):
```

Comment:
I think that we should have the signature:

``def rank(self, method='average', axis=0, numeric_only=None, na_option='keep', ascending=True, pct=False)``

this will be more consistent with other methods.